lib/src/components/NxPolicyViolationIndicator/NxPolicyViolationIndicator.scss

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-text-helpers';
+
+.nx-policy-violation-indicator {
+  align-items: center;
+  background-color: #fff;
+  border: 1px solid;
+  border-radius: $nx-border-radius;
+  display: flex;
+  height: 30px;
+  margin: 0 0 $nx-spacing-xs $nx-spacing-md;
+  max-width: 860px;
+  width: fit-content;
+}
+
+.nx-policy-violation-indicator__badge {
+  border-top-left-radius: $nx-border-radius;
+  border-bottom-left-radius: $nx-border-radius;
```

Comment:
if the `.nx-policy-violation-indicator` had `overflow: hidden` this wouldn't be necessary.  That would be the preferred solution as doing it this way creates a gap:
![image](https://user-images.githubusercontent.com/3630091/100640966-f0563780-3304-11eb-8469-775ad6f59986.png)

Reply:
> if the `.nx-policy-violation-indicator` had `overflow: hidden` this wouldn't be necessary. That would be the preferred solution as doing it this way creates a gap:
> ![image](https://user-images.githubusercontent.com/3630091/100640966-f0563780-3304-11eb-8469-775ad6f59986.png)

Yes, I never remember that: https://github.com/sonatype/sonatype-react-shared-components/pull/229/commits/85320ea38d108b6339d28c5b4e3dba332ad1c6c4